apps/builder/app/routes/_ui.error.tsx

```diff
@@ -25,9 +22,57 @@ export const meta: MetaFunction<typeof loader> = () => {
+  },
+  {
+    message:
+      "Unknown error Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+    description:
+      "Unknown error Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+  },
+  {
+    message:
+      "UnknownerrorLoremIpsumissimplydummytextoftheprintingandtypesettingindustry.LoremIpsumhasbeentheindustry'sstandarddummytexteversincethe1500s,whenanunknownprintertookagalleyoftypeandscrambledittomakeatypespecimenbook.Ithassurvivednotonlyfivecenturies,butalsotheleapintoelectronictypesetting,remainingessentiallyunchanged.Itwaspopularisedinthe1960swiththereleaseofLetrasetsheetscontainingLoremIpsumpassages,andmorerecentlywithdesktoppublishingsoftwarelikeAldusPageMakerincludingversionsofLoremIpsum.",
+    description:
+      "UnknownerrorLoremIpsumissimplydummytextoftheprintingandtypesettingindustry.LoremIpsumhasbeentheindustry'sstandarddummytexteversincethe1500s,whenanunknownprintertookagalleyoftypeandscrambledittomakeatypespecimenbook.Ithassurvivednotonlyfivecenturies,butalsotheleapintoelectronictypesetting,remainingessentiallyunchanged.Itwaspopularisedinthe1960swiththereleaseofLetrasetsheetscontainingLoremIpsumpassages,andmorerecentlywithdesktoppublishingsoftwarelikeAldusPageMakerincludingversionsofLoremIpsum.",
+  },
+];
+
 export const loader = async ({ request }: LoaderFunctionArgs) => {
   preventCrossOriginCookie(request);
 
+  // We leave it here because production errors in remix has a different behavior
+  // https://wstd.dev:5173/error?development-error=0
+  // https://wstd.dev:5173/error?development-error=1
+  // ...
+  const url = new URL(request.url);
+  if (url.searchParams.has("development-error")) {
+    const index = Number(url.searchParams.get("development-error"));
+    if (index >= 0 && index < developmentErrors.length) {
+      throw new Response(JSON.stringify(developmentErrors[index]), {
```

Comment:
this looks like an error page testing tool that's in production instead of a storybook story

Reply:
Because storybook is not remix.

Reply:
I mean the content of the page could be rendered in a story without remix dependency, I did it for dashboard for example

Reply:
The only way now to test how it behave in production is publishing it

Reply:
I thought you are tesitng layout as well, I meant to test layout etc and have a snapshots

Reply:
don't care too much though, just looks a bit weird because its tests but they are in prod